main.rb

```diff
@@ -0,0 +1,144 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+# rubocop:disable all
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_books
+    @books.each do |book|
+      puts "Title: #{book.title} , Author: #{book.author}"
+    end
+  end
+
+  def all_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      nil
+    end
+
+    print 'Age:'
+    age = gets.chomp
+
+    print 'Name:'
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
+
+        Student.new(age, name, parent_permission)
```

Comment:
-  [ ] When I try to create a student it throws an argument error. Kindly fix this issue. :pray: 

![Screenshot from 2021-11-20 00-04-53](https://user-images.githubusercontent.com/64894927/142692384-ee97fcf3-6edc-4c19-b673-804abb041ce3.png)

Reply:
Done ✔️

Reply:
Great job! :+1: :+1: :+1: